views/js/qtiCreator/widgets/interactions/associateInteraction/states/Question.js

```diff
@@ -36,7 +46,10 @@ define([
        var interaction = this.widget.element;
 
         $form.html(formTpl({
-            shuffle : !!interaction.attr('shuffle')
+            shuffle : !!interaction.attr('shuffle'),
+            enabledFeatures: {
+                shuffleChoices: features('taoQtiItem/creator/interaction/associate/property/shuffle')
```

Comment:
issue: it should be `features.isVisible`, no?

Reply:
Yes, that's right, thanks for noticing👍